### PR TITLE
add karpenter to landscape

### PR DESCRIPTION
**Pull request by jwcesign**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other [guidelines](https://github.com/cncf/landscape#new-entries)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?

### CODE REVIEW

**Comment by pacoxu** [comment]
It's surprising that Karpenter is not included in the landscape.

/lgtm
/cc @donaldliu @caniszczyk

File: hosted_logos/karpenter.svg
Comment (gazal-k) [inline hosted_logos/karpenter.svg:1]: This logo does not look like the official one 🤔

Comment (gazal-k) [reply hosted_logos/karpenter.svg:1]: Perhaps one of the Karpenter maintainers can help with one?

Comment (jwcesign) [reply hosted_logos/karpenter.svg:1]: Please check again @gazal-k

**Comment by jwcesign** [comment]
cc @jonathan-innis @ellistarn @engedaam @njtran

**Pull request merged**